smoke/IotEdgeQuickstart/Quickstart.cs

```diff
@@ -78,9 +78,9 @@ public async Task RunAsync()
                         }
                     }
                 }
-                catch(Exception)
+                catch(Exception e)
                 {
-                    Console.WriteLine("** Oops, there was a problem. We'll stop the IoT Edge runtime, but we'll leave it configured so you can investigate.");
+                    Console.WriteLine($"** Oops, there was a problem. We'll stop the IoT Edge runtime, but we'll leave it configured so you can investigate. /n Exception: {e}");
```

Comment:
`/n` should be `\n`, but anyway you should split it into a separate `WriteLine` call.